msi.gama.headless/src/msi/gama/headless/runtime/Application.java

```diff
@@ -41,8 +41,11 @@
 import org.osgi.framework.Bundle;
```

Comment:
This commit name is misleading. It's actually resolving the conflict from new headless. Github handle this by merging the upstream to the PR.